### ISSUE TITLE
(Fresh install of) CLion + Rust plugin causes freeze at "Loading components"

### ISSUE THREAD

**Issue by avwie** [intellij-rust/intellij-rust#4407]
## Environment

* **IntelliJ Rust plugin version: 0.2.105.2133-192**
* **Rust toolchain version: rustup 1.19.0 (2af131cf9 2019-09-08) / rust 1.37.0**
* **IDE name and version: CLion 2019.2**
* **Operating system: Mac OSX 10.14.6**

## Problem description
CLion refused to load after update of CLion to 2019.2 and Rust plugin to latest version. It froze at "Loading components". Deinstall and reinstall of IDE didn't solve the problem.

## Logging
In ~/Library/Logs/CLion thread-dumps are found. They are all the same. It seems like a thread-lock or something.
[threadDump-20190919-155050.txt](https://github.com/intellij-rust/intellij-rust/files/3631566/threadDump-20190919-155050.txt)

**Comment by njam** [comment]
I experience the same with "IDEA".
I wasn't sure it's caused by the Rust plugin. Will look for thread dumps the next time it happens.

---
```
=== About ===
Build version: IntelliJ IDEA 2019.2.2 Build #IU-192.6603.28 September 6, 2019
Java version: 11.0.3+12-b304.56amd64
Operating System: Linux (5.2.14-arch2-1-arch, amd64)
JVM version: OpenJDK 64-Bit Server VM JetBrains s.r.o

=== Plugins ===
Custom plugins: [GitLab Projects (2.0.1), Protobuf Support (0.13.0), Makefile support (1.6.1), CSV Plugin (2.7.0), HashiCorp Terraform / HCL language support (0.7.6), BashSupport (1.7.13.192), Handlebars/Mustache (192.6262.9), JS GraphQL (2.2.0), PHP (192.6603.42), Go (192.6603.23.335), PHP Remote Interpreter (192.6262.16), Vue.js (192.6262.27), PHP Docker (192.5728.26), Python (2019.2.192.6603.23), Ruby (2019.2.20190829), Toml (0.2.0.25), Rust (0.2.105.2133-192)]
Disabled plugins:[]
```

**Comment by garyttierney** [comment]
FYI, I was able to resolve this by:

```sh
> $ rm -R .idea/
> $ rm *.iml
```

And re-importing the project.

**Comment by avwie** [comment]
@garyttierney 
Ah yes, that appears to work. Thanks!

**Comment by njam** [comment]
Here's another thread dump:
[threadDump-20190920-085402.txt](https://github.com/intellij-rust/intellij-rust/files/3634558/threadDump-20190920-085402.txt)

**Comment by Undin** [comment]
Looks like it's indeed deadlock in platform https://youtrack.jetbrains.com/issue/IDEA-220475
The bug is already fixed in 2019.3 and will be fixed in next 2019.2 releases
